cmd/config.go

```diff
@@ -255,7 +255,6 @@ func initConfig() {
 		if err == errConfigNotFound {
 			// Config file does not exist, we create it fresh and return upon success.
 			logger.FatalIf(newConfig(objAPI), "Unable to initialize minio config for the first time")
-			logger.Info("Created minio configuration file successfully at " + getConfigDir())
```

Comment:
there should be some success message, this looks as if there is a failure. ' We can simply say, `Created minio configuration successfully` .